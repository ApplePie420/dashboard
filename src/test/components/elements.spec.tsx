import { render } from "@testing-library/react";
import {
  ListContainer,
  Headline,
  SubHeadline,
  ItemList,
  Item,
  Button,
} from "../../components/elements";

it("Tests Lists", () => {
  const { asFragment } = render(
    <ListContainer>
      <ItemList>
        <Item>Test</Item>
      </ItemList>
    </ListContainer>,
  );

  expect(asFragment).toMatchSnapshot();
});

it("Tests Headline", () => {
  const { asFragment } = render(<Headline>Test</Headline>);

  expect(asFragment).toMatchSnapshot();
});

it("Tests SubHeadline", () => {
  const { asFragment } = render(<SubHeadline>Test</SubHeadline>);

  expect(asFragment).toMatchSnapshot();
});

it("Tests Button", () => {
  const { asFragment } = render(<Button>Test</Button>);

  expect(asFragment).toMatchSnapshot();
});
