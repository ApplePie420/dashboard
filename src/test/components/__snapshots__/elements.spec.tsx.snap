// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tests Button 1`] = `[Function]`;

exports[`Tests Headline 1`] = `[Function]`;

exports[`Tests Lists 1`] = `[Function]`;

exports[`Tests SubHeadline 1`] = `[Function]`;
